docs/src/model_setup/buoyancy_and_equation_of_state.md

```diff
@@ -220,19 +220,23 @@ To simulate gravitational accelerations that don't align with the vertical (`z`)
 we wrap the buoyancy model in
 `Buoyancy()` function call, which takes the keyword arguments `model` and `gravity_unit_vector`,
 
-```jldoctest buoyancy
+```jldoctest buoyancy; filter=(r".*tartarus.*")
 julia> θ = 45; # degrees
 
 julia> g̃ = (0, sind(θ), cosd(θ));
 
 julia> model = NonhydrostaticModel(; grid, 
                                    buoyancy=Buoyancy(model=BuoyancyTracer(), gravity_unit_vector=g̃), 
                                    tracers=:b)
+┌ Warning: The meaning of `gravity_unit_vector` changed in version 0.80.0.
+│ In versions 0.79 and earlier, `gravity_unit_vector` indicated the direction _opposite_ to gravity.
+│ In versions 0.80.0 and later, `gravity_unit_vector` indicates the direction of gravitational acceleration.
+└ @ Oceananigans.BuoyancyModels ~/builds/tartarus-16/clima/oceananigans/src/BuoyancyModels/buoyancy.jl:48
```

Comment:
For some reason things are getting stuck at this doctest after I add the filter option. I think this filter is correct based on [the docs](https://github.com/JuliaDocs/Documenter.jl/blob/master/docs/src/index.md) and a few examples of it being used that I found.

@navidcy do you have any idea of how to solve this?

Reply:
EDIT: I tried something else. Let's wait to see if that works before trying anything new.

Reply:
The docs get stuck... I'll try to run locally to see if doctests pass.